zetaclient/zetacore_observer.go

```diff
@@ -160,7 +163,18 @@ func (co *CoreObserver) startSendScheduler() {
 						for idx, send := range sendList {
 							params := send.GetCurrentOutTxParam()
 							if params.ReceiverChainId != c.ChainId {
-								log.Warn().Msgf("mismatch chainid: want %d, got %d", c.ChainId, params.ReceiverChainId)
+								co.logger.ZetaChainWatcher.Warn().Msgf("mismatch chainid: want %d, got %d", c.ChainId, params.ReceiverChainId)
+								continue
+							}
+							currentHeight := uint64(bn)
+							nonce := params.OutboundTxTssNonce
+							outTxID := fmt.Sprintf("%s-%d-%d", send.Index, params.ReceiverChainId, nonce) // should be the outTxID?
```

Comment:
What does the comment mean?

Reply:
This one was pretty old when some one unsure about the best pattern of the ID to be used in `OutTxProcessorManager`. I can rephrase the comment in case we need to improve it in future.

Reply:
I don't think we have a value `outTxID`, do you mean `outTxHash`? I think it could be used think we can safely assume it is unique

Reply:
Sorry, it's an identifier that only used in zetaclient. Here is how outTxID is calculated: https://github.com/zeta-chain/node/blob/btc-keysign-schedule/zetaclient/chainclient.go#L24
![image](https://github.com/zeta-chain/node/assets/34498985/32cef5c9-f875-4b75-8f22-a77815e5d0c7)